docs/learn/networks/testnets.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Testnets
+description: The Development and Testnet implementations for Research purposes. Participate in future developments here.
+---
+
+
+# Test- and Developmentnetworks
+
+The IOTA Foundation and the IOTA community members operate several Development Networks in which different stages of the IOTA protocol and new applications can be tested and optimized.
+This is a playground for developers and researchers. 
+The assets used in those Testnets are just Testnet-Tokens without any real value.
+
+
+## IOTA 2.0 Decentralized DevNet
+
+This is the implementation of the upcoming fully decentralized IOTA 2.0 Protocol. 
+The network here does not use or need a coordinator anymore. Utilizes Testnet iota tokens without value to fine-tune, test, and optimize the upcoming development stage of the IOTA Protocol together with the IOTA community. This network is composed of Goshimmer nodes. While many nodes in the 2.0 Devnet are run and maintained by the Iota Foundation, anyone can contribute by running their own nodes. Having a larger network is beneficial for testing newly researched and developed concepts and ideas, catching bugs, and ironing out finished components.
+
+- **[API: https://nodes.nectar.iota.cafe/](http://nodes.nectar.iota.cafe/info)**
+- **[Explorer](https://v2.iota.org/explorer)**
+
+## IOTA 1.5 Comnet
+
+Comnet is a 100% community driven testnetwork. It can be used as a testnetwork for your IOTA 1.5 apps and it is also used to test the newest Versions of Hornet or Bee and future IOTA 1.5 upgrades.
```

Comment:
```suggestion
Comnet is a 100% community-driven test network. It can be used as a test network for your IOTA 1.5 apps and it is also used to test the newest Versions of Hornet or Bee and future IOTA 1.5 upgrades.
```